code/__DEFINES/traits.dm

```diff
@@ -670,3 +670,5 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_MAGNETIC_ID_CARD "magnetic_id_card"
 /// Traits granted to items due to their chameleon properties.
 #define CHAMELEON_ITEM_TRAIT "chameleon_item_trait"
+/// Trait from [/datum/component/rust]
+#define RUSTY_TRAIT "rust_trait"
```

Comment:
These are trait *sources*, not actual traits.

Move it up.

Reply:
:clown_face: